src/cargo/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![deny(unused)]
 #![cfg_attr(test, deny(warnings))]
+#![feature(rc_unique)]
```

Comment:
Cargo currently builds on stable, and I'd personally very much like to keep it that way, so can this avoid using unstable features for now?